### PR TITLE
Update new-pssession.action.ps1

### PR DESCRIPTION
**Pull request by dgoldman-msft**
Added functionality to use port for session

### CODE REVIEW

File: PSModuleDevelopment/internal/buildActions/new-pssession.action.ps1 [hunk 1]
```diff
@@ -30,12 +31,13 @@
 }
 
 $params = @{
-	Name	    = 'new-pssession'
-	Action	    = $action
+	Name        = 'new-pssession'
```
Comment (FriedrichWeinmann) [inline PSModuleDevelopment/internal/buildActions/new-pssession.action.ps1:34]: Get your hands off my tabs!

File: PSModuleDevelopment/internal/buildActions/new-pssession.action.ps1 [hunk 2]
```diff
@@ -30,12 +31,13 @@
...
+	Action      = $action
```
Comment (FriedrichWeinmann) [inline PSModuleDevelopment/internal/buildActions/new-pssession.action.ps1:35]: Tabs, Tabs, Tabs

File: PSModuleDevelopment/internal/buildActions/new-pssession.action.ps1 [hunk 3]
```diff
@@ -30,12 +31,13 @@
...
 	Description = 'Establish a PSSession to a target computer and provide it as an artifact'
 	Parameters  = @{
 		ComputerName   = 'The Computer to connect to'
-		VMName		   = 'The virtual machine to which to connect to via the HyperV VM Bus'
+		Port           = 'Port you want to connect too'
```
Comment (FriedrichWeinmann) [inline PSModuleDevelopment/internal/buildActions/new-pssession.action.ps1:39]: `too` should probably be `to`

**Review by FriedrichWeinmann — CHANGES_REQUESTED** [review]
Tabs are ok-ish, but that double-o is a menace
;)

(Demo Review with black humor for teaching Dave the tool, not an actual review)

File: PSModuleDevelopment/internal/buildActions/new-pssession.action.ps1 (hunk 3 above)
Comment (dgoldman-msft) [reply PSModuleDevelopment/internal/buildActions/new-pssession.action.ps1]: Fixed!

**Review by FriedrichWeinmann — APPROVED** [review]

**Pull request merged**